specs/lib/readResourcesSpec.js

```diff
@@ -115,14 +115,27 @@ describe('readResources', () => {
 
     it('rejects when loading resource outside of the resource directory when secure is true', async () => {
         const gltf = readGltf(boxTexturedSeparate2Path);
-        gltf.images[0].uri = '../cesium.png';
+        gltf.images[0].uri = '../box-textured-binary-separate/cesium.png'; // Outside resourceDirectory
```

Comment:
Why was the relative uri changed? Does `'../cesium.png';` still trigger an error?

Reply:
It was because the image didn't exist, so it would trigger a different error, as this test wasn't even testing the `secure` flag. It was throwing an error for not specifying a `resourceDirectory`.

I'm removing `secure` so this test will go away anyway.